src/components/timecard/edit-shift-hours/EditShiftHours.jsx

```diff
@@ -175,6 +204,50 @@ const EditShiftHours = ({
     );
   };
 
+  const hasShiftMovedFromTimecard = (startDate, endDate) => {
+    const startDayTime = dayjs(timecardDate).startOf('day');
+    const endDayTime = dayjs(timecardDate).endOf('day');
+    const currentDayStart = new Date(startDayTime).getTime();
+    const currentDayEnd = new Date(endDayTime).getTime();
```

Comment:
Are these 00:00 and 23:59? Could this be simplified?

Reply:
Simplified dates to use less variables and easier to read comparisons